mobile/lib/utils/tuple.dart

```diff
@@ -0,0 +1,7 @@
+/// A immutable pair or 2-tuple
+class Pair<T1, T2> {
```

Comment:
So Dart has a new record type next release (2.19) or so. I believe it can replace this custom class, correct?

If so, we can put a TODO as a reminder here to refactor later when the release hit.

https://github.com/dart-lang/language/blob/master/accepted/future-releases/records/records-feature-specification.md

Reply:
done